.env

```diff
@@ -1,4 +1,5 @@
 FAASM_VERSION=0.5.14
+FAASM_CLI_IMAGE=faasm/cli:0.5.14
```

Comment:
Following the `Quick Start` running `docker-compose up -d` without this variable in the `.env` file would fail.